### PR TITLE
Update benchmark-tidb-using-tpcc.md

### PR DESCRIPTION
**Pull request by SE-Bin**
### What is changed, added or deleted? (Required)
Update benchmark-tidb-using-tpcc.md

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: benchmark/benchmark-tidb-using-tpcc.md [hunk 1]
```diff
@@ -70,22 +70,45 @@ ant
...
 
-之后可以通过下面的命令来启动 TiDB：
+之后可以通过修改 `{tidb_deploy_path}/scripts/run_tidb.sh` 启动脚本，加入 `numactl` 来启动 TiDB ：
 
 {{< copyable "shell-regular" >}}
 
 ```shell
-nohup taskset -c 0,2,4,6,8,10,12,14,16,18,20,22,24,26,28,30,32,34,36,38 bin/tidb-server && \
-nohup taskset -c 1,3,5,7,9,11,13,15,17,19,21,23,25,27,29,31,33,35,37,39 bin/tidb-server
+$ cat run_tidb.sh
```
Comment (TomShawn) [inline benchmark/benchmark-tidb-using-tpcc.md:85]: @SE-Bin 请将命令分开写。

Comment (SE-Bin) [reply benchmark/benchmark-tidb-using-tpcc.md]: 这个是文件的内容不是命令，已经修改了代码块的类型

File: benchmark/benchmark-tidb-using-tpcc.md [hunk 2]
```diff
@@ -70,22 +70,42 @@ ant
...
+    -P 4111 \
+    --status="10191" \
+    --advertise-address="172.16.4.53" \
+    --path="172.16.4.10:2490" \
+    --config=conf/tidb.toml \
+    --log-slow-query="/home/damon/deploy/tidb1-1/log/tidb_slow_query.log" \
+    --log-file="/home/damon/deploy/tidb1-1/log/tidb.log" 2>> "/home/damon/deploy/tidb1-1/log/tidb_stderr.log"
 ```
 
+>注意：直接修改 run_tidb.sh 可能会被覆盖。因此在生产环境中，如有绑核需求，建议使用 Tiup 绑核
```
Comment (lilin90) [inline benchmark/benchmark-tidb-using-tpcc.md:107]:
```suggestion
> **注意：**
>
> 直接修改 `run_tidb.sh` 可能会被覆盖，因此在生产环境中，如有绑核需求，建议使用 TiUP 绑核。
```

**Pull request merged**